ci/cached-builds/registries.conf

```diff
@@ -0,0 +1,6 @@
+# https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md
+
+# prevent this kyverno error
+#  Failed to pull image "bitnami/kubectl:1.26.4": reading manifest 1.26.4 in quay.io/bitnami/kubectl: unauthorized: access to the requested resource is not authorized
+unqualified-search-registries = [ "docker.io" ]
+short-name-mode = "enforcing"
```

Comment:
i only saw this when I was doing something with kyverno (I don't intend to push that), but it looks like a good default; kyverno only has images in docker.io, but with the previous defaults somehow an attempt to pull from quay.io was made; let's stop that